commitizen/commands/commit.py

```diff
@@ -81,8 +81,13 @@ def _prompt_commit_questions(self) -> str:
 
         message = cz.message(answers)
         message_len = len(message.partition("\n")[0].strip())
-        message_length_limit = self.arguments.get("message_length_limit", 0)
-        if 0 < message_length_limit < message_len:
+
+        message_length_limit = self.arguments.get(
+            "message_length_limit",
+            self.config.settings.get("message_length_limit", None),
+        )
+
+        if message_length_limit is not None and message_len > message_length_limit:
```

Comment:
```suggestion
        if message_length_limit and message_len > message_length_limit:
```

this can check both 0 and None